app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::ProjectsController < ApplicationController
+
+  def search
+    term = params[:term]
+    tags = ActiveSupport::JSON.decode params[:tags]
+    results = []
+    projects = Project.query(term, tags)
+    projects.each_with_index do |x, i|
```

Comment:
Unused block argument - `i`. If it's necessary, use `_` or `_i` as an argument name to indicate that it won't be used.